src/components/RawInput/index.tsx

```diff
@@ -15,7 +14,7 @@ export interface RawInputProps<K> extends Omit<React.HTMLProps<HTMLInputElement>
     /**
     * input name
     */
-    name: K;
+    name?: K;
```

Comment:
We should not do this, it will break typings for useForm

Reply:
We should be able to use inputs without a name

Reply:
When we use Generic K, it can also be undefined because we haven't defined K not to be undefined

Reply:
The eslint-config says otherwise. It can be undefined but we have to put it like `name:={undefined}`

Reply:
Anyway, reverting back. Let's make name required prop